charts/images.yaml

```diff
@@ -229,7 +229,7 @@ images:
 - name: fluent-bit-plugin-installer
   sourceRepository: github.com/gardener/logging
   repository: eu.gcr.io/gardener-project/gardener/fluent-bit-to-loki
-  tag: "v0.36.2"
+  tag: "v0.37.1"
```

Comment:
You have to update the version also for `loki-curator` and `telegraf`, don't you?